brainstormforce/astra
# AST-1426 - Used HTML title tooltip and removed extra tooltip options & customizations

### Description
- Used HTML title tooltip for icon style switcher mode

### Types of changes
- Refactored code - non-breaking change 

### How has this been tested?
- Check switch type icon 
- Check with both switch style

### Checklist:
